### DISCUSSION TITLE
Block multiple subscribers

### DISCUSSION THREAD

**Discussion by skyEnrique** [ant-media/Ant-Media-Server#6069] [Q&A]
Is there a way we can block multiple subscribers using ams rest? Currently we are using
`/v2/broadcasts/{id}/subscribers/{sid}/block/{seconds}/{type}` but the problem is it can only block 1 subscriber at a time.

**Comment by USAMAWIZARD** [comment]
Hi @skyEnrique ,
here is the api  documentation https://antmedia.io/rest/#/BroadcastRestService/blockSubscriber
it does not support using multiple subscriber id in a single rest call , you will need to call the rest api twice with different ids

![image](https://github.com/ant-media/Ant-Media-Server/assets/47350008/53de5016-4d95-46ce-aea0-8741941724c6)
